core/src/test/java/org/arquillian/smart/testing/report/SmartTestExecutionReportUsingPropertyTest.java

```diff
@@ -0,0 +1,102 @@
+package org.arquillian.smart.testing.report;
+
+import java.io.File;
+import net.jcip.annotations.NotThreadSafe;
+import org.arquillian.smart.testing.Configuration;
+import org.arquillian.smart.testing.TestSelection;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.RestoreSystemProperties;
+import org.junit.experimental.categories.Category;
+
+import static java.util.Arrays.asList;
+import static org.arquillian.smart.testing.Configuration.SMART_TESTING;
+import static org.arquillian.smart.testing.report.SmartTestExecutionReport.SMART_TESTING_REPORT_DIR;
+import static org.arquillian.smart.testing.report.SmartTestExecutionReport.SMART_TESTING_REPORT_NAME;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Category(NotThreadSafe.class)
```

Comment:
We should have default values for report directory etc. Users should only have a way to configure it in exceptional cases (convention over configuration). With this we also don't need to mark this test as not thread safe, as it will not touch system properties at all.